pkg/ai/noopai.go

```diff
@@ -16,7 +17,7 @@ type NoOpAIClient struct {
 	model    string
 }
 
-func (c *NoOpAIClient) Configure(token string, model string, language string) error {
+func (c *NoOpAIClient) Configure(token string, model string, language string, baseURL string) error {
```

Comment:
I do agree this is growing, but again, I hold off from any change not in scope with this PR - I think a follow up for a generic config or move to functional style would be needed here.